.eslintrc.js

```diff
@@ -62,6 +62,12 @@ module.exports = {
 		'testing-library/no-unnecessary-act': 'off',
 		'@typescript-eslint/no-empty-function': 'off',
 		'@typescript-eslint/no-var-requires': 'off',
+		'@wordpress/i18n-text-domain': [
+			'error',
+			{
+				allowedTextDomain: 'woocommerce-gateway-stripe',
+			},
+		],
```

Comment:
`@wordpress/i18n-text-domain` was reporting a _lot_ of issues. I chose to specify our expected text domain, but we could also keep to the defaults.